layouts/default.vue

```diff
@@ -1,6 +1,10 @@
 <template>
-  <div>
+  <div
+    class="group/layout"
+    :data-with-container="true"
```

Comment:
What is this used for?

Reply:
The new banner component must should show its content at the same position that our layout : at the edge of the screen if we have a fluid layout or at the current container position if the page is using a container.
We can add the information as data attributes and [target it with CSS](https://github.com/datagouv/front-end/pull/197/files#diff-1e0715102e48517e4d89049836cd321633ffffd1e8f2d9957f29cb86a4733b7fR12) to provide the correct class only when required.